### PR TITLE
Update vagrant environment to Fedora 34

### PR DESCRIPTION
**Pull request by LenkaSeg**
Should solve issue #1127

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -2,18 +2,19 @@ alembic>=1.3.1, <2.0.0
...
-pytoml>=0.1.18, <1.0.0
 semver>=2.8.1, <3.0.0
-social-auth-app-flask>=1.0.0, <2.0.0
 social-auth-app-flask-sqlalchemy>=1.0.1, <2.0.0
+social-auth-app-flask>=1.0.0, <2.0.0
+sqlalchemy>=1.3.17, <2.0.0
 sseclient>=0.0.26, <1.0.0
 straight.plugin>=1.5.0, <2.0.0
-sqlalchemy>=1.3.17, <2.0.0
+toml==0.10.2
```
Comment (Zlopez) [inline requirements.txt:17]: Don't use exact versions, this could fail if version is not available in Fedora yet
```suggestion
toml>=0.10.2, <1.0.0
```

File: requirements.txt [hunk 2]
```diff
@@ -2,18 +2,19 @@ alembic>=1.3.1, <2.0.0
...
 webargs>=8.0.0, <9.0.0
+werkzeug==0.16.1
```
Comment (Zlopez) [inline requirements.txt:19]: Same as above
```suggestion
werkzeug>=0.16.1, <1.0.0
```

File: requirements.txt [hunk 3]
```diff
@@ -2,18 +2,19 @@ alembic>=1.3.1, <2.0.0
...
+flask>=1.1.4, <3.0.0
 jinja2>=2.11.2, <4.0.0
 ordered-set>=3.1, <5.0.0
 python-dateutil>=2.8.0, <3.0.0
-pytoml>=0.1.18, <1.0.0
 semver>=2.8.1, <3.0.0
-social-auth-app-flask>=1.0.0, <2.0.0
 social-auth-app-flask-sqlalchemy>=1.0.1, <2.0.0
+social-auth-app-flask>=1.0.0, <2.0.0
+sqlalchemy>=1.3.17, <2.0.0
```
Comment (Zlopez) [inline requirements.txt:14]: Why the changes in ordering?

**Review by Zlopez — CHANGES_REQUESTED** [review]
Two small changes.

Could you also add to commit message, why the requirements were changed? Something like "pytoml is now toml"  etc.

File: requirements.txt (hunk 3 above)
Comment (LenkaSeg) [reply requirements.txt:14]: I ran `:sort` to sort stuff alphabetically, so it is quicker to find the keywords. When I add a new item to requirements, I usually write it in the top and then run `sort`.

Comment (abitrolly) [reply requirements.txt:14]: But `flask` should be placed before `flask-login`, so that `:sort:` is doing wrong.

Comment (Zlopez) [reply requirements.txt:14]: @abitrolly Yes, I would assume the same. that `flask` should before `flask-login`

Comment (LenkaSeg) [reply requirements.txt:14]: I think in case there would be whitespaces after the name, the sort would work correctly.

Comment (LenkaSeg) [reply requirements.txt:14]: Should I add whitespaces so the sort works or reorder it manually and keep it without spaces?

File: requirements.txt (hunk 2 above)
Comment (Zlopez) [reply requirements.txt:19]: The werkzeug is not direct dependency it could be removed altogether

File: requirements.txt (hunk 3 above)
Comment (Zlopez) [reply requirements.txt:14]: I never used `sort` on requirements file, but it sounds like a good idea. I wonder what will happen if we run `black` on requirements.txt

Comment (Zlopez) [reply requirements.txt:14]: It can't be parsed by `black` unfortunately

Comment (Zlopez) [reply requirements.txt:14]: @LenkaSeg Let's use the manual ordering for this PR and if you want, you can create a separate PR for changing the requirements file to be sortable by `sort`

Comment (LenkaSeg) [reply requirements.txt:14]: Ok, returning it back.

**Review by Zlopez — DISMISSED** [review]
This looks good to me :+1:

**Comment by Zlopez** [comment]
One more thing, could you just update the commit message to describe the changes?

**Comment by LenkaSeg** [comment]
I updated the commit message => pytoml is now toml.
